Add Ddd.jmxfetch.<integration-name>.enabled to java docs (#20480)
DataDog/documentation: content/en/tracing/trace_collection/library_config/java.md

**Default**: `8125`<br>
StatsD port to send JMX metrics to. If you are using Unix Domain Sockets, input 0.

`dd.jmxfetch.<integration-name>.enabled`
: **Environment Variable**: `DD_JMXFETCH_<INTEGRATION_NAME>_ENABLED`<br>
**Default**: `false`<br>

> JMX integration to enable (e.g. kafka, activemq).

Reviewer: ```suggestion
JMX integration to enable (for example, Kafka or ActiveMQ).
```
